### PR TITLE
Hide doc item for deprecated function

### PR DESCRIPTION
**Pull request by Lucretiel**

### CODE REVIEW

**Comment by kornelski** [comment]
That's not just docs - this is a breaking change for users who used this method under this path. It's not worth causing breakage like that.

**Comment by Lucretiel** [comment]
What? No it isn't, the item is still usable. It doesn't make the item inaccessible under this, it just hides it from rustdoc.

**Comment by Lucretiel** [comment]
It's no different than the doc(hidden) on the function defintion itself:

https://github.com/3Hren/msgpack-rust/blob/00eeadfd60ae78cb0cbb0207c40efe9a2836697a/rmp-serde/src/decode.rs#L894-L900

**Comment by kornelski** [comment]
Oh, sorry — I've misread the code.

File: rmp-serde/src/lib.rs [hunk 1]
```diff
@@ -68,9 +68,12 @@ use std::str::{self, Utf8Error};
 use serde::{Deserialize, Serialize};
 use serde::de;
 
-pub use crate::decode::{Deserializer, from_read, from_read_ref, from_slice};
+pub use crate::decode::{Deserializer, from_read, from_read_ref};
 pub use crate::encode::{Serializer, to_vec, to_vec_named};
 
+#[doc(hidden)]
+pub use crate::decode::from_slice
```
Comment (kornelski) [inline rmp-serde/src/lib.rs:75]:
```suggestion
pub use crate::decode::from_slice;
```

**Pull request merged**